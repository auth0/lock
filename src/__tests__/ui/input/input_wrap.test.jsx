import React from 'react';

import { expectComponent } from 'testUtils';
import InputWrap from '../../../ui/input/input_wrap';

describe('InputWrap', () => {
  const defaultProps = {
    after: <span>after</span>,
    isValid: true,
    name: 'name'
  };

  test('renders correctly with the `after` prop', () => {
    expectComponent(<InputWrap {...defaultProps} />).toMatchSnapshot();
  });
});
