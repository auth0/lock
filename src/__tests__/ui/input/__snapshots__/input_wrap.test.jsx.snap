// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWrap renders correctly with the \`after\` prop 1`] = `
<div
  className="auth0-lock-input-block auth0-lock-input-name"
>
  <div
    className="auth0-lock-input-wrap"
  />
  <span>
    after
  </span>
</div>
`;
